include/core_sys.h

```diff
@@ -13,18 +13,18 @@ enum ButtonMode {
 };
 
 typedef struct CoreSys {
-    UnkFuncPtr_02017798 unk_00;
-    void * unk_04;
-    UnkFuncPtr_02017798 unk_08;
-    void * unk_0C;
+    Callback mainCallback;
+    void * mainCallbackData;
+    Callback hblankCallback;
+    void * hblankCallbackData;
```

Comment:
Pull the names on these pointer-vars back a character.

```suggestion
    Callback mainCallback;
    void *mainCallbackData;
    Callback hblankCallback;
    void *hblankCallbackData;
```